creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -217,6 +221,10 @@ module.exports = async function (
         throw new Error(
           `Cannot sync - imported data is not contiguous. Local max clock val = ${localMaxClockVal} and imported min clock val ${fetchedClockRecords[0].clock}`
         )
+      } else if (maxClockRecordId !== fetchedLatestClockVal) {
+        throw new Error(
+          `Cannot sync - imported data is not consistent. Imported max clock val = ${fetchedLatestClockVal} and imported max ClockRecord val ${maxClockRecordId}`
+        )
```

Comment:
i prob misunderstood but i thought this check was meant to be inside export and trigger a retry? cc @dmanjunath it also makes sense here though

Reply:
@SidSethi this is in addition to that, not instead of. here's the export code https://github.com/AudiusProject/audius-protocol/pull/3480/files#diff-603c68a5c307de706249f27b45a7499857db7ed1402ac39b6aefa0ca0fae3e1cR126-R129

Reply:
Ya Dheeraj asked if I could also add a check here